docs/source/install/index.rst

```diff
@@ -124,11 +124,12 @@ for the same purpose.
       cmake -DUSE_CUDA=ON ..
       make -j4
 
-Finally, install the Python binding.
+Finally, upgrade the python depedencies, then install Python binding.
 
 .. code:: bash
 
    cd ../python
+   pip3 install --user  numpy networkx scipy
```

Comment:
i don't quite understand why we need this. these three libraries are installed by DGL automatically.

Reply:
Cython is not required by default since windows may meet problem when installing cython

Reply:
u are right
i reran without this line and it worked fine,  removing this line